assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/workers/cl/worker.yaml.tmpl

```diff
@@ -315,10 +315,12 @@ storage:
           :FORWARD ACCEPT [0:0]
           :OUTPUT ACCEPT [0:0]
           -A INPUT -i lo -j ACCEPT
-          -A INPUT -p tcp --dport 22 -j ACCEPT
-          -A INPUT -p tcp --dport 179 -j ACCEPT
-          -A INPUT -p tcp --dport 10250 -j ACCEPT
-          -A INPUT -p tcp --dport 10256 -j ACCEPT
+          %{ for management_cidr in management_cidrs ~}
+          -A INPUT -s ${management_cidr} -p tcp --dport 22 -j ACCEPT
+          %{~ endfor }
```

Comment:
I don't think ssh is needed for workers before they are running calico (out of band console can be used if needed). What do you think?

Reply:
It seems reasonable, though opinionated, so it should be documented. If calico fails to start for some reason, it makes it more difficult to debug. What do you think @kinvolk/team-cloud?

Reply:
Hm, maybe port 22 should also be allowed from `node_private_cidr`?

Reply:
- IMO we want to be able to SSH into workers at all times, and especially during failures.
- I don't think we want to allow SSH from `node_private_cidr`. This increases the nodes' attack surface (very theoretically) but mainly I don't find it useful. My guess is you're suggesting this to allow using a bastion host (right?), in which case I would include the bastion's `/32` address in `management_cidrs` because it's a management CIDR.